src/lib.rs

```diff
@@ -42,7 +40,18 @@ pub use transport::{Connection, ConnectionChannelsBuilder, Packet};
 
 pub type ConnectionHandle = u32;
 
-pub struct NetworkingPlugin;
+#[derive(Default)]
+pub struct NetworkingPlugin {
+    link_conditioner: Option<LinkConditionerConfig>,
+}
+
+impl NetworkingPlugin {
```

Comment:
Since we have a `with_link_conditioner` constructor, I'd rather have `new()` constructor instead `Default` impl.

It's not like `NetworkingPlugin` will be a part of some other `Default` derived struct.

Reply:
It might actually be better to make every field in `NetworkingPlugin` public and mantain `Default` unless we want to have configuration in a separate resource

Reply:
Do you mean this pattern:
```rust
.add_plugin(NetworkingPlugin {
    link_conditioner: Some(link_conditioner),
    ..Default::default(),
})
```
If yes, I agree - it is even better.